tests/unit/Actions.spec.ts

```diff
@@ -220,10 +220,9 @@ describe('Actions Filing Functionality', () => {
     expect(mockBuildFiling).toHaveBeenCalled()
     expect(mockBuildFiling).toHaveReturned()
 
-    // verify redirection
-    const baseUrl = 'myhost/cooperatives/NR 1234567'
-
-    expect(window.location.assign).toHaveBeenCalledWith(baseUrl)
+    // also verify event emission
+    const events = wrapper.emitted('goToDashboard')
+    expect(events.length).toBe(1)
```

Comment:
"Actions" now emits this event instead of redirecting. ("App" does the actual redirection.)